src/register/register.ts

```diff
@@ -146,10 +316,49 @@ export class Register {
         })
       );
 
-      Register.registers[register].text = text;
-    }
+      // Harmonize newline character
+      text = text.replace(/\r\n/g, '\n');
```

Comment:
These lines only apply when we read from the system clipboard. The problem is vscode automaticaly transforms `\n` to `\r\n` when your EOL is set to `CRLF`, so when we're on a windows machine the EOL is `\r\n` if i would paste that vscode transforms it to `\r\r\n` and you've got two linebreaks in the editor. 
My first idea was to get the current EOL setting from the active editor but i didn't found the vscode API, you can set it but i don't found where i can read the setting??? 
As i can't go that way, my next idea was check the OS if we're on windows replace the newline char with the vscode-vim internal newline ('\n' we use it everywhere), but then i thought safe me the check and replace it everytime because on windows it fixes the double newline and on unix systems it causes no harm because there is no `\r\n` in the clipboard.

What do you think, is it better to add the OS check so it is more obvious that it should only apply to windows machines?